vagrant/spec/masters.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: cluster.k8s.io/v1alpha1
+kind: MachineSet
+metadata: 
+  name: master
+spec: 
+  replicas: 1
+  template: 
+    metadata: 
+    spec: 
+      providerConfig: 
+        value: 
+          apiVersion: vagrantMachineSet/v1alpha1
+          kind: vagrantMachineSet
+          box: "ubuntu/xenial64"
+          cpus: 2
+          memory: 2048
```

Comment:
These seem like good CPU/mem defaults

One thing that's notable is that this is a base-image change.
The previous workflow uses `bento/ubuntu-17.10` which is a Chef community maintained box for Artful Aardvark along with the kubernetes xenial repos.

Reply:
That's a simple example/default configuration.
You can change this one or better create your own cluster api specification to pass to `kubeadm playground` command